launcher/image/test/test_discover_signatures.yaml

```diff
@@ -4,7 +4,7 @@ substitutions:
   '_CLEANUP': 'true'
   '_VM_NAME_PREFIX': 'discover-signatures'
   '_ZONE': 'us-west1-a'
-  '_WORKLOAD_IMAGE': 'us-west1-docker.pkg.dev/confidential-space-images-dev/cs-integ-test-images/basic-test:latest'
+  '_WORKLOAD_IMAGE': 'us-west1-docker.pkg.dev/confidential-space-images-dev/cs-integ-test-images/ipc/happypath:latest'
```

Comment:
We should also have a test for the default token to ensure signed container works with it as well

Reply:
Done. I created a separate test for ODA and signed container, and keep this test for the default token.